assets/js/components/notifications/ZeroDataStateNotifications/index.js

```diff
@@ -134,25 +134,34 @@ export default function ZeroDataStateNotifications() {
 	}
 
 	let gatheringDataTitle;
+	let gatheringDataWaitTime;
```

Comment:
Can we simplify it by setting the default value as `72`?
```suggestion
	let gatheringDataWaitTime = '72';
```